pkg/kubectl/resource/visitor.go

```diff
@@ -222,7 +222,7 @@ func (v *PathVisitor) Visit(fn VisitorFunc) error {
 		if !v.IgnoreErrors {
 			return err
 		}
-		fmt.Fprintf(os.Stderr, "error: unable to load file %q: %v\n", v.Path, err)
+		fmt.Fprintf(os.Stderr, "error: %v\n", err)
```

Comment:
That's fine. Long term I'd like more consistent error handling in kubectl, as this seems like a one off patch and there are still deeper problems. Thanks for explanation.